02LessonTwo/homework/Homework.html

```diff
@@ -0,0 +1,101 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title></title>
+  <link rel="stylesheet" href="css/style.css">
+</head>
+<body>
+  <div id="header">
+    <h1>Larry Gutierrez</h1>
+    <h1>EST.1991<h1>
+  </div>
+  <div id="left">
+    <table>
+      <th>
+        <h2>About Me</h2>
+        <p>Whats up guys im Larry also know in the streets as Bird you guys can most probably make the
+        connection there. Lived all my life in San Antonio so you know im a spurs fan.
+        A entrepenuer at heart always curious about how thing come into existense, how they work
+        and grow. I like to think of my self as a discplined, honest and loving individual. Family
+        is everything to me. I love chinesse food like seriously i can probaly eat all the time.
+       </p>
+     </th>
+   </table>
+     <h3>Hobbies<h3>
+     <Ul>
+       <li>Outdoors</li>
+       <li>traveling</li>
+       <li>futbol</li>
+       <li>boxing</li>
+     </Ul>
+
+      <H3>Thing i dont like.</H3>
+      <select name="cars">
+  <option value="v">Volvo</option>
+  <option value="A">Saab</option>
+  <option value="fiat">waking up early</option>
+  <option value="terror">ISIS</option>
+  <option value="audi">alergies</option>
+  <option value="audi">cold</option>
+  <option value="audi">loosing</option>
+  <option value="audi">Audi</option>
+  <option value="audi">Audi</option>
+  <option value="audi">Audi</option>
+</select>
+</div>
+  <div class="topright">
+    <h3>Favorite websites</h3>
+    <table cellspacing="30px">
+      <td><a href="https://www.ted.com/talks"><img src="http://www.brainscape.com/blog/wp-content/uploads/2011/10/ted-talks-language-learning.png"alt="ted" style="width:100px;height:100px;border-radius:5px"/></td>
+      <td><a href="http://www.vice.com/en_us"><img src="https://upload.wikimedia.org/wikipedia/en/thumb/c/c0/Vice_logo.svg/1024px-Vice_logo.svg.png"alt="ted" style="width:100px;height:100px;border-radius:5px"/></td>
+      <td><a href="https://www.tumblr.com/"><img src="https://secure.assets.tumblr.com/images/logo_page/img_logo_34465d_2x.png"alt="ted" style="width:100px;height:100px;border-radius:5px"/></td>
+      <td><a href="https://twitter.com/"><img src="https://g.twimg.com/Twitter_logo_blue.png"alt="ted" style="width:100px;height:100px;border-radius:5px"/></td>
+
+      </td>
+    </table>
```

Comment:
This is a much more appropriate use of the `<table>` tag. I really like the table spacing and use of borders.